### PR TITLE
docs(cli): Improve CLI documentation structure

### PR DESCRIPTION
**Pull request by santiagogaray**
Hi @chriswmackey,

Nice... second repo without issue on my side! Please take a look when you get a chance!

### CODE REVIEW

**Comment by chriswmackey** [comment]
Thanks, @santiagogaray . Will review soon.

File: docs/conf.py [hunk 1]
```diff
@@ -232,6 +228,248 @@
...
+#                    },
+#    library_name_2: {CLI_filename_3: group_name_3,
+#                     CLI_filename_4: group_name_4
+#                    }
+# }
+ht_cli_file_group = {
+    'honeybee_radiance': {'mtx': 'mtxop',
+                          'postprocess': 'post_process'
+                          }
+}
```
Comment (chriswmackey) [inline docs/conf.py:273]: As I mentioned in the other PR, could we remove this?

**Review by chriswmackey — COMMENTED** [review]
Just one comment to resolve and then we can merge.

File: docs/conf.py (hunk 1 above)
Comment (santiagogaray) [reply docs/conf.py]: Done!

**Review by chriswmackey — APPROVED** [review]
LGTM! Merged!

If you are looking for the next repos that need updating, both the ladybug-comfort and ladybug repos have CLIs that would be good to put into the new format.

**Pull request merged**